tests/Responses/Expected/product_attribute/show_response_after_partial_update.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": @integer@,
+    "code": "mug_material",
+    "type": "text",
+    "values": [],
+    "position": 0,
+    "createdAt": "@string@.isDateTime()",
+    "updatedAt": "@string@.isDateTime()",
+    "translations": {
+        "de": {
```

Comment:
The behavior is correct. If you want to see that, the translation for `de_CH` was hanged go to `product_attribute.yml` and set there the translation for `de_CH` (because now it has the translation for `de`).